### PR TITLE
feat: made several visual updates to the import github modal and populated the resource datatable

### PR DESCRIPTION
**Pull request by tangopapatime**
# Description
Update the github import modal with the suggest visual enhancements
Populated resource manager datatable

Resolves #(issue)

https://github.com/DARPA-ASKEM/Terarium/issues/1164
https://github.com/DARPA-ASKEM/Terarium/issues/1168

which are related to https://github.com/DARPA-ASKEM/Terarium/issues/1144

Sorry merged main and get things back to create the video :P

### CODE REVIEW

**Comment by mwdchang** [comment]
Food for thoughts: 

I am a bit skeptical about all the z-index changes, because the numbers are relative to the context where they are set and they are not compared against each other in a absolute sense. 

It may be better if we have a layout schematic and we can use that as a reference rather than tweaking z-indices all over the place. @jamiewaese-uncharted

**Comment by tangopapatime** [comment]
> Food for thoughts:
> 
> I am a bit skeptical about all the z-index changes, because the numbers are relative to the context where they are set and they are not compared against each other in a absolute sense.
> 
> It may be better if we have a layout schematic and we can use that as a reference rather than tweaking z-indices all over the place. @jamiewaese-uncharted

absolutely.  I knew out of all changes there might a comment about this.  So, maybe I did it wrong, but it was the only way I was able to get the `DropDown` list to work within the modal.  I just scaled the indexes so the order should be in theory preserved, at least for the elements who's z-index was predefined.

File: packages/client/hmi-client/src/components/widgets/tera-import-github-file.vue [hunk 1]
```diff
@@ -99,13 +101,13 @@
 									:key="index"
 									@click="previewTextFile(content)"
 								>
-									<input
-										type="checkbox"
-										:id="content.name"
-										:value="content"
+									<Checkbox
```
Comment (dvince2) [inline packages/client/hmi-client/src/components/widgets/tera-import-github-file.vue:104]: Out of curiosity, what is the difference here between a `Checkbox` and a `input type="checkbox`?

**Review by dvince2 — APPROVED** [review]

File: packages/client/hmi-client/src/components/widgets/tera-import-github-file.vue (hunk 1 above)
Comment (tangopapatime) [reply packages/client/hmi-client/src/components/widgets/tera-import-github-file.vue:104]: one is a `primevue` component and the latter is the native version.

**Comment by tangopapatime** [comment]
[updates.webm](https://github.com/DARPA-ASKEM/Terarium/assets/98172806/364679e9-b2cf-41aa-aa7c-935c468fc265)

**Pull request merged**